docs/main.adoc

```diff
@@ -18,7 +18,7 @@ image::images/digitaliseringsdirektoratet.png[width=50%, pdfwidth=30vw]
 ++++
 <div style="position: fixed; top: 40%; left: 50%; transform: translateX(-50%); background-color: #ffffff; border: 1px solid black; padding: 15px; box-shadow: 15px 15px 15px red; opacity: 0.85">
   <p> <b>NB! Denne versjonen av DCAT-AP-NO er under utfasing og vil ikke bli vedlikeholdt (inkl. ev. kjente feil/mangler).</b> <br>
-   Vi oppfordrer sterkt til å bruke <a href=https://data.norge.no/specification/dcat-ap-no>den til enhver tid gjeldende versjon av DCAT-AP-NO</a>. </p>
+   Vi oppfordrer sterkt til å bruke <a href=https://data.norge.no/specification/dcat-ap-no>den til enhver tid gjeldende versjon av DCAT-AP-NO</a>. </b> </p>
```

Comment:
Mangler ikke denne en startende bold-tag, `<b>`, at den er ubalansert ?

I tillegg, om vi skal gi uttrykk for en viktig beskjed bør vi bruke `<strong>` for denne beskjeden.

Reply:
Kan også være lurt å skrive ending tag for line-breaken: `<br />`

Reply:
OK `<b>` -> `<strong>`
Ikke `<br />` (se https://www.w3schools.com/tags/tag_br.asp)